<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileShredderClass</class>
 <widget class="QMainWindow" name="FileShredderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1078</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1078</width>
    <height>605</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FileShredder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgba(68,70,84,255);
}

QScrollBar:vertical {
            border: 0px solid #999999;
            background:  rgba(247, 247, 247,150);
            width:10px;    
            margin: 0px 0px 0px 0px;
        }
        QScrollBar::handle:vertical {         
       
            min-height: 100px;
          	border: 0px solid red;
			border-radius: 4px;
			background-color: black;
        }
        QScrollBar::add-line:vertical {       
            height: 0px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
        }
        QScrollBar::sub-line:vertical {
            height: 0 px;
            subcontrol-position: top;
            subcontrol-origin: margin;
        }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="FileListView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>961</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(198, 198, 198, 0.6);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
    </property>
   </widget>
   <widget class="QFrame" name="OptionsFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>961</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#OptionsFrame {
background-color: rgba(53,54,64,0.8);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="RemoveFilesCheckBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: white;
border: none</string>
     </property>
     <property name="text">
      <string>Remove Files After Wipe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WipeButton">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(32,33,35,255);
	color: white;
	border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:hover {
   background-color: #575965;
	color: white;
    border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:pressed {
   background-color:#b1b9bb;
	color: white;
}
</string>
     </property>
     <property name="text">
      <string>Wipe Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ChooseFilesButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum 20 files for wipe.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(32,33,35,255);
	color: white;
	border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:hover {
   background-color: #575965;
	color: white;
    border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:pressed {
   background-color:#b1b9bb;
	color: white;
}

</string>
     </property>
     <property name="text">
      <string>Choose Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearScreenButton">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(32,33,35,255);
	color: white;
	border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:hover {
   background-color: #575965;
	color: white;
    border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:pressed {
   background-color:#b1b9bb;
	color: white;
}
</string>
     </property>
     <property name="text">
      <string>Clear Screen</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="PassesSpinBox">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum 10 passes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: white;
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="NumberOfPassesLabel">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: white;
border: none</string>
     </property>
     <property name="text">
      <string>Number Of Passes</string>
     </property>
    </widget>
    <widget class="QFrame" name="SpinBoxFrame">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(32,33,35,255);
border-radius: 15px;
border-width: 2px;
border-radius: 15px;
padding: 4px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelWipeButton">
     <property name="geometry">
      <rect>
       <x>318</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(32,33,35,255);
	color: white;
	border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:hover {
   background-color: #575965;
	color: white;
    border-radius: 15px;
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border-color: black;
	padding: 4px;
}

QPushButton:pressed {
   background-color:#b1b9bb;
	color: white;
}
</string>
     </property>
     <property name="text">
      <string>Cancel Wipe</string>
     </property>
    </widget>
    <zorder>SpinBoxFrame</zorder>
    <zorder>RemoveFilesCheckBox</zorder>
    <zorder>WipeButton</zorder>
    <zorder>ChooseFilesButton</zorder>
    <zorder>ClearScreenButton</zorder>
    <zorder>PassesSpinBox</zorder>
    <zorder>NumberOfPassesLabel</zorder>
    <zorder>CancelWipeButton</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileShredder.qrc"/>
 </resources>
 <connections/>
</ui>
